primer/view_components
# Upgrade Primitives + prep for v8 release

- Bump to latest version
- Update all file paths to `dist`
- Remove the fallback plugin (we don't need it anymore)

⚠️ I think we should hold off on merging this until I merge https://github.com/github/github/pull/320584
